src/tests/api/e2eHappyPath.test.ts

```diff
@@ -17,6 +17,8 @@ import {
 
 
 describe("E2E Happy Path Tests", () => {
+	jest.setTimeout(60000);
```

Comment:
Howcome?

Reply:
I added this because the tests were timing out but actually that turned out to be because of mutliple test harnesses. Removed [](url)